tests/StlTests.cpp

```diff
@@ -11,7 +11,7 @@ struct dummy
 {
     char value;
 };
-
+using namespace wil;

```

Comment:
suspicious namespace alias.

Reply:
our coding guidelines say never create namespace aliases like this for wil and std. 

I understand string literal require an aliases as discussed in the closed comments. I assume you should use that here to demonstrate the best usage.